src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutBase.cs

```diff
@@ -73,10 +73,7 @@ internal void Hide(bool canCancel)
 			Closed?.Invoke(this, EventArgs.Empty);
 		}
 
-		internal protected virtual void Close()
```

Comment:
Why is this necessary?

Reply:
need to be implemented in derived classes ..

I just removed the useless and dangerous 
throw new NotImplementedException("This method has no base class implementation and must be overridden in a derived class.");

since base.close() would crash

note that i havent cleaned Open() and CreatePresenter either.. but they should be cleaned..

flyoutbase should actually be abstract and these should be abstract method or virtual but empty. 

here where close() is used.. 
internal void Hide(bool canCancel)
		{
			if (!_isOpen)
			{
				return;
			}

			if (canCancel)
			{
				var closing = new FlyoutBaseClosingEventArgs();
				Closing?.Invoke(this, closing);
				if (closing.Cancel)
				{
					return;
				}
			}

			Close();
			_isOpen = false;
			Closed?.Invoke(this, EventArgs.Empty);
		}